<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Popup</class>
 <widget class="QDialog" name="Popup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>375</width>
    <height>315</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#frame {
background-color: qlineargradient(spread:pad, x1:0.489, y1:0, x2:0.5, y2:0.994318, stop:0 rgba(203,64,18, 1), stop:1 rgba(133,32,1, 1));
border: 2px solid #DD3401;
border-radius: 40px;
}


QPushButton {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
}

QLabel {
color: white;
}

QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}

QComboBox {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}

QPushButton:hover { 
    color: black;
} 

QPushButton:pressed { 
    color: lightgray; 
    padding-top: -15px; 
    padding-bottom: -17px; 
}

QPushButton[enabled=&quot;false&quot;] {
	background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #1C1C2B, stop: 0.1 #3A3A4E, stop: 0.49 #3A3A46, stop: 0.5 #5F5F77, stop: 1 #808095);
}


QMessageBox  {
background-color: qlineargradient(spread:pad, x1:0.489, y1:0, x2:0.5, y2:0.994318, stop:0 rgba(203,64,18, 1), stop:1 rgba(133,32,1, 1));
border: 2px solid #DD3401;
border-radius: 40px;
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btnPopSave">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPopClose">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>250</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>close</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnPopAddAtr">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add Attribute</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Icons/Assets/Icons/symbol140.png</normaloff>:/Icons/Assets/Icons/symbol140.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>291</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: transparent;</string>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>289</width>
         <height>69</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2"/>
      </widget>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>271</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lEObject">
         <property name="toolTip">
          <string>Object Name.</string>
         </property>
         <property name="placeholderText">
          <string>Object</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lE_UL">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Upper Left point coordinates</string>
         </property>
         <property name="placeholderText">
          <string>UL coordinate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lE_LR">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Lower Right point coordinates</string>
         </property>
         <property name="placeholderText">
          <string>LR coordinate</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btnPopLayoutReset">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>140</y>
        <width>21</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="toolTip">
       <string>Reset Layout.</string>
      </property>
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;clear Layout from all widget.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string extracomment="Reset Layout"/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Icons/Assets/Icons/reset-or.png</normaloff>:/Icons/Assets/Icons/reset-or.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
